src/Hero.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+export default function Hero() {
+  return (
+    <div className="hero">
+      <hr />
+      <div className="textHeroContainer">
+        <h1>A perfect CYF Experience </h1>
+        <p>
+          <i>
+            The Darenth is in an ‘Area of Outstanding Natural Beauty’ with
+            spectacular views over a manicured landscape. Our Gallery and
+            Pavilion are ideal for indoor and outdoor ceremonies and The
+            Ballroom is perfect for exclusive celebrations.
+          </i>
+        </p>
+        <p>
+          <i>
+            We are located in Sevenoaks, close to the M25 and within 30 minutes
+            of Chislehurst, Bromley, Tunbridge Wells and Maidstone. We have
+            onsite accommodation and a Church and train station within walking
+            distance.
+          </i>
+        </p>
+        <p>
+          <i>
+            Our experienced and friendly team will ensure that your special day
+            is detailed, fabulous and forever memorable.
+          </i>
+        </p>
+      </div>
+      <hr />
+    </div>
+  );
+}
```

Comment:
Lots of simple components, this is a great way to do it, as its so easy to see whats going on!